ruby-gem/Rakefile

```diff
@@ -36,14 +35,31 @@ def build
 end
 
 task :build do
```

Comment:
Noted, but I will do this in a separate PR. The goal of this PR is to get the `rake build` running more or less exactly as it was before.